packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -1505,7 +1506,7 @@ export class ApexDebug extends LoggingDebugSession {
     }
     try {
       response.success = false;
-      const errorObj = JSON.parse(error);
+      const errorObj = extractJsonObject(error);
```

Comment:
@AnanyaJha these changes still show a blank modal when running Apex Interactive debugger without the right user perms. Am I missing something when running this PR locally ?

Reply:
whoa okay i'm not seeing that, let me look into it. are you on the latest version of the cli?  and the same set up as nick's post? https://github.com/forcedotcom/salesforcedx-vscode/issues/2005

Reply:
Okay I'm actually not able to repro it, still getting this message

Reply:
Ananya and I synced on this and turns out the issue I'm seeing is specific to VSCode insiders. We'll merge these changes and create a follow up PR to address insiders.